app/src/main/java/io/github/trojan_gfw/igniter/MainActivity.java

```diff
@@ -555,6 +569,17 @@ public void onClick(DialogInterface dialog, int which) {
         });
     }
 
+    @UiThread
+    private void updatePortInfo(long port) {
+        currentProxyPort = port;
+        if (INVALID_PORT == port) {
```

Comment:
How about limit the port range to [0, 65535]

Reply:
Reasonable